### ISSUE TITLE
[BUG] inspectdb dosent work with leatest backend version

### ISSUE THREAD

**Issue by Fleapse** [jayvynl/django-clickhouse-backend#58]
**Describe the bug**
inspectdb dosent work with leatest backend version

on version 1.1.1 - it definitely worked
(at that time I had a different version of the clickhouse server without a cluster, maybe that’s the problem)

**To Reproduce**
run `python manage.py inspectdb --database=clickhouse`
and get

```
# This is an auto-generated Django model module.
# You'll have to do the following manually to clean this up:
#   * Rearrange models' order
#   * Make sure each model has one field with primary_key=True
#   * Make sure each ForeignKey and OneToOneField has `on_delete` set to the desired behavior
#   * Remove `managed = False` lines if you wish to allow Django to create, modify, and delete the table
# Feel free to rename the models, but don't rename db_table values or field names.
from django.db import models
```
without any models

**Expected behavior**
models should be generated

**Versions**

- Python version.  3.11
- Django version. 4.1.9
- Django clickhouse backend version. 1.1.3
- Server clickhouse version. 23.9.1.1854

**Comment by jayvynl** [comment]
Thanks for the report, I will try to fix it.

**Comment by jayvynl** [comment]
It's fixed and will be released as 1.1.4. Now you can install from the main branch.

I have implemented an improved version of inspectdb #61. Add `clickhouse_backend` to INSTALL ED_APPS to use it.
